src/cljs/athens/events.cljs

```diff
@@ -267,7 +285,8 @@
 (reg-event-fx
   :transact
   (fn [_ [_ datoms]]
-    {:transact! datoms}))
+    {:transact! datoms
```

Comment:
Probably want to make sure everything use `:transact` now instead of `:transact!` directly, so that the not-synced UI is shown.